data/data.json

```diff
@@ -9146,19 +9146,19 @@
    "weaveNode": "weaveworks/weave-kube:2.8.1",
    "weaveCni": "weaveworks/weave-npc:2.8.1",
    "podInfraContainer": "rancher/mirrored-pause:3.6",
-   "ingress": "rancher/nginx-ingress-controller:nginx-1.2.1-rancher1",
+   "ingress": "rancher/nginx-ingress-controller:nginx-1.2.0-rancher1",
```

Comment:
The data.json is pulled in during CI and should not be modified directly, instead the changes should be merged in KDM and then pulled through here.

Reply:
Revert back the changes [c1e3dc3](https://github.com/rancher/rke/pull/2963/commits/c1e3dc35396f7419efed3f51373f0ea613b9352d)